### PR TITLE
Environment Variable Management API + Thanos Offset

### PR DESCRIPTION
**Pull request by mbolt35**
* Add more convenient environment variable API and convention to add new environment variables. 
* Add a `thanos` package which contains specific configuration data related to thanos.

### CODE REVIEW

File: pkg/costmodel/costmodel.go [hunk 1]
```diff
@@ -51,6 +52,7 @@ const (
 	remoteEnabled  = "REMOTE_WRITE_ENABLED"
 	thanosEnabled  = "THANOS_ENABLED"
 	thanosQueryUrl = "THANOS_QUERY_URL"
+	thanosOffset   = "THANOS_QUERY_OFFSET"
```
Comment (nikovacevic) [inline pkg/costmodel/costmodel.go:55]: Is this being used?

File: pkg/env/costmodelenv.go [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+// GetConfigPath returns the environment variable value for ConfigPathEnvVar which represents the cost
+// model configuration path
+func GetConfigPath() string {
+	return Get(ConfigPathEnvVar, "")
+}
+
+// GetConfigPath returns the environment variable value for ConfigPathEnvVar which represents the cost
+// model configuration path
+func GetConfigPathWithDefault(defaultValue string) string {
```
Comment (nikovacevic) [inline pkg/env/costmodelenv.go:111]: Do you think we could get away with one function called `GetConfigPath()` that defaulted to some constant in the package whose value is `"/models/"`? The reasoning being that the only places where `GetConfigPath()` is currently used are commented-out, and everywhere this function is used, the same default is given.

(I don't feel strongly about this, just playing devil's advocate.)

File: pkg/costmodel/costmodel.go (hunk 1 above)
Comment (mbolt35) [reply pkg/costmodel/costmodel.go:55]: Ah, crap - looks like all of these got added back.

File: pkg/env/costmodelenv.go (hunk 2 above)
Comment (mbolt35) [reply pkg/env/costmodelenv.go:111]: You're probably right, but was just trying to set a convention for any env vars in the future that would need a reasonable default passed in.

**Pull request merged**